src/tinybox.vue

```diff
@@ -202,6 +202,30 @@ export default {
         window.removeEventListener('keyup', this.keyup);
       }
     },
+    /*
+     * Center the thumbnails' scrollbar to the clicked image
+     */
+    index(index) {
+      var elements, el, thumbs;
```

Comment:
Codacy found an issue: [Expected variable declaration to be on a new line.](https://app.codacy.com/manual/NickKaramoff/vue-tinybox/pullRequest?prid=5742748)